py3status/modules/volume_status.py

```diff
@@ -23,10 +23,12 @@
         (default None)
     device: Device to use. Defaults value is backend dependent
         (default None)
+    device_is_speaker: Should this control the speakers (True) or the microphone (False)?
+        (default True)
     format: Format of the output.
-        (default '♪: {percentage}%')
+        (default '[\?if=device_is_speaker ♪|😮]: {percentage}%')
```

Comment:
After looking at this code some now, I think `is_input` would be the best way to go with this... and to flip the boolean. When users want to enable input, they should have `is_input = True` in their config rather than `is_output = False` (to enable input instead of disabling output).

`is_source` / `is_sink` # pulse naming
`is_capture` / `is_playback` # alsa naming
`is_input` / `is_output` # generic naming as f--- and we should go with this?